docs/crafted-osx.org

```diff
@@ -21,11 +21,11 @@ to be set:
 
 | Keybinding | Meaning                          |
 |------------+----------------------------------|
-| ⌘-W        | (upper case W) delete-frame      |
-| ⌘-}        | tab-bar-switch-to-next-tab       |
-| ⌘-{        | tab-bar-switch-to-prev-tab       |
-| ⌘-t        | tab-bar-new-tab                  |
-| ⌘-w        | (lower case w) tab-bar-close-tab |
+| ⌘-W       | (upper case W) delete-frame      |
```

Comment:
formatting looks off here.

Reply:
Oh yeah, I see it now, too. The `⌘` causes the automatic table formatting in org to be a bit wonky, but I can see it when looking at the raw org in the github web interface. Thanks for pointing that out.